doc/.wordlist.txt

```diff
@@ -37,3 +37,4 @@ Jira
 VM
 YAML
 CephFS
+cephbr
```

Comment:
This is not a word and shouldn't be here. ;)

Reply:
fixed